lib/improver/tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -444,11 +391,9 @@ def test_realizations_and_source_realizations_fails(self):
     def test_multiple_realizations(self):
         """Test when the cube has a realization dimension."""
         cube = set_up_cube(num_realization_points=4)
-        radii = 14400
+        radii = 5600
```

Comment:
This test passes, so I assume this value is right, but where does it come from? I got 5760 by replicating what the ens adjustment was doing:
```
0.8 * sqrt(radii ** 2 / num_realization_points)
0.8 * sqrt(14400 ** 2 / 4) = 5760
```
Help me.

Reply:
We no longer apply any adjustments to the radii based on realizations. So I took the value from the test below (of 5600) and just checked that if I added realizations I got the same numbers, to prove that it isn't automatically adjusting the radii based on realizations.